async-inference/Async-Inference-Walkthrough-SageMaker-Python-SDK.ipynb

```diff
@@ -0,0 +1,652 @@
+{
```

Comment:
<p>Style - preferred to use the language "Deletes" instead of "Tears down"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>

Reply:
Done